src/export-page/data/constants.js

```diff
@@ -5,3 +5,4 @@ export const EXPORT_STAGES = {
   COMPRESSING: 2,
   SUCCESS: 3,
 };
+export const SUCCESS_DATE_FORMAT = 'MM/DD/yyyy';
```

Comment:
Can you get value from `DATE_FORMAT` instead create a new one

Reply:
`DATE_FORMAT` contain `dd` instead of `DD` and write day in string format:
it's bug describes in this ticket https://youtrack.raccoongang.com/issue/2U-606